pom.xml

```diff
@@ -1180,7 +1180,7 @@
             <dependency>
                 <groupId>com.h2database</groupId>
                 <artifactId>h2</artifactId>
-                <version>1.4.199</version>
+                <version>1.4.200</version>
```

Comment:
Let us know if you were able to figure out the root cause of the issue we discussed internally.

Reply:
Ya I found the issue. H2db version 1.4.200 only support the start date from "1890-09-30". If i am give above that this date then test cases will pass.
Reference link : https://github.com/h2database/h2database/issues/2261
Currently i am trying to upgrade the version as 2.2.220, which has no vulnerabilities.

Reply:
Good find. :D